protected/migrations/m190401_105351_alter_table_dataset_column_dataset_size.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class m190401_105351_alter_table_dataset_column_dataset_size extends CDbMigration
```

Comment:
If the down() functions in the 3 migration scripts are not implemented then it would help if it is explained why as a comment in the code. For example, see [here](https://github.com/gigascience/gigadb-website/blob/develop/protected/migrations/m181108_105104_widen_user_password_column.php).